pkg/annotation/annotation.go

```diff
@@ -0,0 +1,199 @@
+package annotation
+
+import (
+	"bufio"
+	"io"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+var (
+	// IgnoreRegexp finds the gocover ignore pattern.
+	// Three kind ignore pattern are supported:
+	//   //+gocover:ignore:all
+	//   //+gocover:ignore:block
+	//   //+gocover:ignore:{number}
+	//
+	// - `//+gocover:ignore:all`
```

Comment:
nit: how about using `//+gover:ignore:file` to exclude the whole file? I feel like the `:all` is too vague